.github/workflows/dockerhub.yml

```
@@ -0,0 +1,46 @@
+
+    - name: Extract Git information
+      id: git-info
+      run: |
+        echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | sed 's/\\//-/g')" >> $GITHUB_ENV
+        echo "COMMIT_SHA=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+
+    - name: Build and Push Docker Image
+      run: |
+        IMAGE_NAME=cognee/cognee
+        TAG_VERSION="${BRANCH_NAME}-${COMMIT_SHA}"
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)